event_handler/requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
```

Comment:
Clever! I don't think I've seen this pattern in a requirements file.  I like it.

Reply:
This line in the noxfile is unnecessary if we expect `requirements-test` to include `requirements`: https://github.com/GoogleCloudPlatform/fourkeys/blob/main/noxfile.py#L61

Should we remove?

Reply:
I’m inclined to leave it: if we were to have a Python script for which the tests don’t have any additional requirements, it would be reasonable for that folder to contain only `requirements`. The overhead of installing `requirements` twice should be negligible.

Reply:
(I feel like it’s also _possible_ that the test requirements would not be a strict superset of prod requirements. E.g. if the app requires gunicorn, but the tests don’t actually make any web requests.)